src/main/java/uk/gov/ons/ctp/integration/rhsvc/RHSvcApplication.java

```diff
@@ -69,4 +75,17 @@ public EventPublisher eventPublisher(final ConnectionFactory connectionFactory)
   public RestExceptionHandler restExceptionHandler() {
     return new RestExceptionHandler();
   }
+
+  /**
+   * Connect to Google Firestore.
+   *
+   * @return a Firestore connection.
+   */
+  @Bean
```

Comment:
As discussed does this need to be outside/away from the FireStoreDataStore that actually uses it?

Reply:
Have move the creation of the Firestore connection into the FirestoreDataStore class. Helps to contain the everything related to Firestore in just one class.